index.bs

```diff
@@ -1779,13 +1779,15 @@ Since the {{MLOperand/[[builder]]}} object is bound by the {{MLGraphBuilder/cons
 
 If an operation supports only a subset of {{MLOperandDataType}}s, the <dfn>allowed data types</dfn> for each of the operation's input operands, including both positional arguments and options, are given as either an explicit list of {{MLOperandDataType}}s, or a constraint that the operand's [=MLOperand/dataType=] must be the <dfn lt="same type as">same as</dfn> the [=MLOperand/dataType=] of another input operand, or <dfn lt="any data type">any</dfn> to allow any {{MLOperandDataType}}.
 
-Implementations may support fewer data types for operands than specified. This can be queried for each operation using the {{MLContext/opSupportLimits()}} method on {{MLContext}} and inspecting the {{MLTensorLimits/dataTypes}} value of the corresponding member for the operation.
+Implementations MAY support fewer data types for operands than specified, but MUST support at least the specified <dfn>minimum allowed data types</dfn>. Support can be queried for each operation using the {{MLContext/opSupportLimits()}} method on {{MLContext}} and inspecting the {{MLTensorLimits/dataTypes}} value of the corresponding member for the operation.
```

Comment:
> MUST support at least the specified <dfn>minimum allowed data types</dfn>

This adds an interesting implication back for ORT as a backend, since data type support varies *per EP*. Though, so long as the CPU EP exists, then fallback should satisfy that expectation (even if not the best performance due to device copies).

(no action expected - resolve me)